client/src/components/SideInfoPanal/SideInfoPanal.jsx

```diff
@@ -8,7 +8,6 @@ import GamesContext from '../../context';
 
```

Comment:
Panal should be Panel. If you hit Cmd+Shift+F you can find every instance of panal in your entire working directory and then replace them.